doc/modules/language-guide/pages/message-inspection.adoc

```diff
@@ -48,17 +48,20 @@ in order to simply dispatch on the name of a function without inspecting its act
 
 NOTE:  Confusingly, a `shared query` function _can_ be called using a regular HTTP call to obtain a certified response: this is why the variant type also includes `shared query` functions.
 
-WARNING: An actor that fails to declare system field `inspect_message` will simply accept all ingress messages.
+WARNING: An actor that fails to declare system field `inspect` will simply accept all ingress messages.
 
-WARNING: `inspect_message` should *not* be used for definitive access control; reliable access control checks can only be performed within the `shared` functions guarded by `inspect_message`. This is because `inspect_message` is executed by a single replica, without consensus and its result could be spoofed by a malicious boundary node.
-A simple, contrived example is a counter actor, that inspects some of its messages in detail, and others only superficially:
+WARNING: System function`inspect` should *not* be used for definitive access control. This is because `inspect` is executed by a single replica, without full consensus, and its result could be spoofed by a malicious boundary node. Reliable access control checks can only be performed within the `shared` functions guarded by `inspect`.
```

Comment:
```suggestion
WARNING: System function `inspect` should *not* be used for definitive access control. This is because `inspect` is executed by a single replica, without full consensus, and its result could be spoofed by a malicious boundary node. Reliable access control checks can only be performed within the `shared` functions guarded by `inspect`.
```